python_modules/dagster/dagster/cli/pipeline.py

```diff
@@ -749,6 +777,8 @@ def get_config_from_args(kwargs):
                     serializable_error_info_from_exc_info(sys.exc_info()).to_string(),
                 )
             )
+    else:
+        check.failed("Should not reach here")
```

Comment:
nit: some context on the condition "unhandled case getting config from kwargs"  or something